cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,19 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+REGISTRY?="gcr.io"
```

Comment:
@thockin thanks for the review.

Without the single quotes around the variable the value is reflected without the quotes.  But, I agree that the literal value should be identical as if the variable was set by user input. Therefore, no quotes.
`[rhallisey@localhost test]$ cat Makefile 
A ?= "a"
B ?= "b"
C ?= "c"

foo:
        echo '$(A)'
        echo $(B)
        echo '$(C)'
[rhallisey@localhost test]$ make C="c"
echo '"a"'
"a"
echo "b"
b
echo 'c'
c
`